src/AccessControlClient.ts

```diff
@@ -5,36 +5,47 @@
 /** @packageDocumentation
  * @module AccessControlClient
  */
-import type {
-  IAccessControlClient,
-  IGroupMembersClient,
-  IGroupsClient,
-  IITwinJobsClient,
-  IMemberInvitationsClient,
-  IOwnerMembersClient,
-  IPermissionsClient,
-  IRolesClient,
-  IUserMembersClient,
-} from "./accessControlTypes";
+
+import type { IAccessControlClient } from "./accessControlClientInterfaces/accessControl";
+import type { IGroupsClient } from "./accessControlClientInterfaces/GroupClient";
+import type { IGroupMemberInvitationClient } from "./accessControlClientInterfaces/GroupMemberInvitationClient";
+import type { IGroupMembersClient } from "./accessControlClientInterfaces/GroupMembersClient";
+import type { IITwinJobsClient } from "./accessControlClientInterfaces/ITwinJobsClient";
+import type { IITwinSharesClient } from "./accessControlClientInterfaces/ItwinSharesClient";
+import type { IMemberInvitationsClient } from "./accessControlClientInterfaces/MemberInvitationsClient";
+import type { IOwnerMembersClient } from "./accessControlClientInterfaces/OwnerMembersClient ";
+import type { IPermissionsClient } from "./accessControlClientInterfaces/PermissionsClient";
+import type { IRolesClient } from "./accessControlClientInterfaces/RolesClient";
+import type { IUserMembersClient } from "./accessControlClientInterfaces/UserMembersClient";
+import { GroupMemberInvitationClient } from './subClients/GroupMemberInvitationsClient';
+import { GroupMembersClient } from "./subClients/GroupMembersClient";
+import { GroupsClient } from "./subClients/GroupsClient";
+import { ITwinJobsClient } from "./subClients/ITwinJobsClient";
+import { ITwinSharesClient } from "./subClients/ItwinShares";
+import { MemberInvitationsClient } from "./subClients/MemberInvitationsClient";
+import { OwnerMembersClient } from "./subClients/OwnerMembersClient";
 import { PermissionsClient } from "./subClients/PermissionsClient";
 import { RolesClient } from "./subClients/RolesClient";
-import { GroupsClient } from "./subClients/GroupsClient";
 import { UserMembersClient } from "./subClients/UserMembersClient";
-import { GroupMembersClient } from "./subClients/GroupMembersClient";
-import { OwnerMembersClient } from "./subClients/OwnerMembersClient";
-import { MemberInvitationsClient } from "./subClients/MemberInvitationsClient";
-import { ITwinJobsClient } from "./subClients/ITwinJobsClient";
 
+/** Client API to access the access control service.
+ * @beta

```

Comment:
I think this should be removed. This class would not be beta? Some on the clients or methods might be beta, but this class I do not think will change.